### PR TITLE
fix: python generated code error when without HTTP headers

### PR DESCRIPTION
**Pull request by wt-goodluck**

### CODE REVIEW

**Review by LinuxSuRen — COMMENTED** [review]
Please update the corresponding e2e test cases as well.

* [add a new case without HTTP header](https://github.com/LinuxSuRen/api-testing/blob/b9533ee974a31dacbf14bc94700eed37d195f847/e2e/code-generator/test-suites/test-suite.yaml#L15)
* trigger the new test case [at here](https://github.com/LinuxSuRen/api-testing/blob/b9533ee974a31dacbf14bc94700eed37d195f847/e2e/code-generator/entrypoint.sh#L18)

File: e2e/code-generator/test-suites/test-suite.yaml [hunk 1]
```diff
@@ -7,9 +7,13 @@ param:
...
-      auth: fake
+  - name: requestWithHeader
+    request:
+      api: /GetSuites
+      method: POST
+      header:
+        auth: fake
+  - name: requestWithoutHeader
+    request:
+      api: localhost:8080/server.Runner/GetSuites
```
Comment (LinuxSuRen) [inline e2e/code-generator/test-suites/test-suite.yaml:18]: `localhost:8080/server.Runner/GetSuites` should be `/GetSuites`

File: pkg/server/remote_server.go [hunk 2]
```diff
@@ -1027,4 +1039,25 @@ func (s *UniqueSlice[T]) GetAll() []T {
 	return s.data
 }
 
+// normalizeAPI function normalizes the provided API address
+func normalizeAPI(api string) (string, error) {
+	if api == "" {
+		return "", nil
+	}
+
+	// 检查是否包含 "://"，如果不包含，则添加默认的协议头（例如 http://）
```
Comment (LinuxSuRen) [inline pkg/server/remote_server.go:1048]: Please write the comment in English.

File: e2e/code-generator/entrypoint.sh [hunk 3]
```diff
@@ -12,9 +12,12 @@ fi
 mkdir -p /root/.config/atest
 mkdir -p /var/data
 
-nohup atest server --local-storage '/workspace/test-suites/*.yaml'&
+nohup atest server --local-storage '/workspace/test-suites/*.yaml' &
 sleep 1
 
+curl http://localhost:8080/server.Runner/RunTest -X POST \
```
Comment (LinuxSuRen) [inline e2e/code-generator/entrypoint.sh:18]: The generated code is not been executed.

Comment (LinuxSuRen) [reply e2e/code-generator/entrypoint.sh:18]: By the way, why using `/server.Runner/RunTest` instead of `/server.Runner/GenerateCode`

Comment (wt-goodluck) [reply e2e/code-generator/entrypoint.sh:18]: Sorry, this was my oversight

**Review by LinuxSuRen — APPROVED** [review]
LGTM thanks for your effort.

**Pull request merged**